modules/phase_field/examples/rigidbodymotion/tests

```diff
@@ -3,27 +3,27 @@
     type = RunApp
     input = 'AC_CH_Multigrain.i'
     cli_args = '--check-input Mesh/uniform_refine=0 Executioner/Adaptivity/initial_adaptivity=0'
+
     # This example uses ShapeElementUserObject which contains a
-    # mooseWarning() that is treated as an error in dbg mode.
-    # Therefore only check this test in opt mode.
-    method = 'OPT'
+    # mooseWarning()
+    executable_pattern = 'phase_field-\w+$'
```

Comment:
I guess I don't understand what this is doing.  Wouldn't this match with any METHOD?